src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +36,24 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={query}
+        onChange={handleSearch}
       />
       <span className="icon is-left">
         <i className="fas fa-magnifying-glass" />
       </span>
+      {query && (
+        <span className="icon is-right" style={{ pointerEvents: 'all' }}>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
let's fix this eslint issue:smiley:
try to give `aria-label` to button, i think it can help